roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -67,8 +69,53 @@ def roi_locations(self):
         return np.array([j['med'] for j in self.stat]).T
 
     @staticmethod
-    def write_segmentation(segmentation_extractor, nwb_file_path):
-        return NotImplementedError
+    def write_segmentation(segmentation_object, save_path):
+        save_path = Path(save_path)
+        if not save_path.is_dir():
+            raise ValueError('provide a folder as save_path')
+        if segmentation_object.__class__.__name__=='MultiSegmentationExtractor':
+            segext_objs = segmentation_object.segmentations
+            if segext_objs.__class__.__name__!='Suite2pSegmentationExtractor':
+                raise ValueError('provide a MultisegmentationExtractor of multiple Suite2pSegmentationExtractor objects')
```

Comment:
I'd prefer an `assert` here

Reply:
> very close!

@bendichter @Saksham20 

I'll have a last round of fixes here and merge